pkg/metrics/metrics.go

```diff
@@ -0,0 +1,93 @@
+/*
+Copyright 2018 Google Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package metrics
+
+import (
+	"flag"
+	"net/http"
+	"strings"
+
+	"github.com/GoogleCloudPlatform/netd/pkg/metrics/collector"
+	"github.com/golang/glog"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+)
+
+type metricsConfig struct {
```

Comment:
if this type is not used anywhere, you can use anonymous type:

```
var mcfg struct {
   enabledCollectors string
   ...
}
```

Reply:
Updated